index.html

```diff
@@ -361,5 +361,36 @@ <h4>Skills</h4>
 
     </section>
 
+    <section class="contact">
+      <div class="contact-text">
+        <p>
+          I'm always interested in hearing about new projects, 
+          so if you'd like to chat please get in touch.
+        </p>
+      </div>
+
+      <form action="" class="contact-form" method="post">
+        <input type="text" name="name" class="contact-form-text" maxlength="30" placeholder="Full name" required>
+        <input type="email" name="email" class="contact-form-email" placeholder="Email address" required>
+        <textarea name="contact" name="message" class="contact-textarea" maxlength="500" placeholder="Enter your text here" required></textarea>
```

Comment:
- [ ] [OPTIONAL] Good job so far! If you remove on of the `name` attribute it would be clean 👍🏼